src/cmdc_tools/datasets/official/IN/data.py

```diff
@@ -12,144 +12,91 @@ class Indiana(DatasetBaseNoDate, CountyData):
-            "DATE": "dt",
-            "COVID_COUNT_CUMSUM": "cases_total",
-            "COVID_DEATHS_CUMSUM": "deaths_total",
-            "COVID_TEST_CUMSUM": "test_total",
-        }
-
-        for _county in county_jsons:
-            _df = pd.DataFrame.from_records(_county["properties"]["VIZ_DATE"]).rename(
-                columns=crename
-            )
-
-            _df["dt"] = pd.to_datetime(_df["dt"])
-            _df["fips"] = int(_county["properties"]["GEOID"])
-
-            dfs.append(_df.loc[:, ["dt", "fips", "cases_total", "deaths_total"]])
-
-        county = pd.concat(dfs, axis=0, ignore_index=True).melt(
-            id_vars=["dt", "fips"], var_name="variable_name", value_name="value"
-        )
-
-        return county
-
-    def _get_hosp(self):
-        url = (
-            "https://www.coronavirus.in.gov/map/"
-            "covid-19-indiana-universal-report-current-public.json"
-        )
-
-        res = requests.get(url)
+        return res
 
+    def _extract_data(self, res):
         # Put into dataframe and dump garbage district... No idea why
-        # it shows up but it isn't an Indiana fips code
+        # it shows up but it isn't an Indiana fips code (18901, the
+        # garbage district, does not show up as of August 26 but will
+        # leave the filter just in case it reappears
         df = pd.DataFrame(res.json()["metrics"]["data"])
         df = df.query("district != '18901'")
 
+        # Variables of interest
         column_names = {
+            # Descriptive
             "date": "dt",
             "district": "fips",
+            # Case/Death/Test variables
+            "m1e_covid_cases_cumsum": "cases_confirmed",
+            "m1e_covid_cases_prob_cumsum": "cases_suspected",
+            "m1e_covid_deaths_cumsum": "deaths_confirmed",
+            "m1e_covid_deaths_prob_cumsum": "deaths_suspected",
+            "m1e_covid_tests_cumsum": "tests_total",
+            # Hospitalization/ICU/Ventilator variables
             "m1a_beds_all_occupied_beds_covid_19_smoothed": "hospital_beds_in_use_covid_total",
             "m2b_hospitalized_icu_supply": "icu_beds_capacity_count",
             "m2b_hospitalized_icu_occupied_covid": "icu_beds_in_use_covid_total",
             "m2b_hospitalized_icu_occupied_non_covid": "icu_beds_in_use_noncovid",
             "m2b_hospitalized_vent_supply": "ventilators_capacity_count",
             "m2b_hospitalized_vent_occupied_covid": "ventilators_in_use_covid_total",
             "m2b_hospitalized_vent_occupied_non_covid": "ventilators_in_use_noncovid",
-            "m1e_covid_tests_cumsum": "tests_total",
-            "m3b_covid_tests_positive_lag": "positive_tests_daily",
         }
-
         df = df.rename(columns=column_names)
 
-        # To datetime and create new variables
+        # Convert date to datetime
         df["dt"] = pd.to_datetime(df["dt"])
+
+        # Create new variables
+        df["cases_total"] = df.eval("cases_confirmed + cases_suspected")
```

Comment:
Are we certain that this isn't double counting anything?

Is the `m1e_covid_cases_cumsum` variable (which we named `cases_confirmed`) for sure cases_confirmed?

Reply:
Yep -- Matches the numbers from [this dashboard](https://www.coronavirus.in.gov/2393.htm) for "Positive Cases"